backend/app/models/blog.js

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
```

Comment:
Good good but add comments explaining what a Schema is and the difference between a database and a collection